cms/static/js/spec/views/pages/course_outline_spec.js

```diff
@@ -624,6 +634,15 @@ define(["jquery", "common/js/spec_helpers/ajax_helpers", "common/js/components/u
                     this.$("#id_time_limit").trigger('focusout');
                 };
 
+                selectPrerequisite = function() {
+                    this.$("#is_prereq").prop('checked', true).trigger('change');
```

Comment:
It is not because we are using some css magic to display a fancy icon based checkbox.